content/en/blog/grpconf-2024-schedule.md

```diff
@@ -0,0 +1,24 @@
+---
+title: gRPConf 2024 Schedule
+date: 2024-07-03
+---
+
+Attention gRPC community!
+
+Mark your calendars for **August 27th, 2024**, as gRPConf returns to the Google Cloud Campus in Sunnyvale, California.
+
+The schedule is live now at: [events.linuxfoundation.org/grpconf/program/schedule](events.linuxfoundation.org/grpconf/program/schedule).
```

Comment:
You want an https:// in that URL. Are you sure that is the link text you want? Seems you could do `[The schedule] is now live!`. Or `live now on [the gRPConf event website].` Or something.

Reply:
Yeah, that's better.